.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
Will delete this file...

Reply:
I'll add the .vscode folder to .gitignore